app/views/measure_conditions/_measure_condition_code_quantity.html.erb

```diff
@@ -21,7 +21,7 @@
       <% else %>
         <td class="govuk-table__cell numerical"><%= condition.document_code %></td>
         <td class="govuk-table__cell">
-          <%= condition.requirement.to_s.html_safe %>
+          <%= condition.certificate_description.presence || condition.requirement.to_s.html_safe %>
```

Comment:
a method in `condition` might be handy, but I am not sure about a potential name for it.

Reply:
This feels quite like a 'view' level concern but in general terms this will all likely get replaced as part of the rework Matt raised last week so I'm keen to stick with the existing pattern for now which at least makes it easy to tailor to different types.